purchase_reception_status_line/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Purchase Reception Status Line",
-    "version": "17.0.1.0.0",
+    "version": "17.0.1.1.0",
```

Comment:
as the module is not yet merged I think you don't need to bump a new version, you can trigger the scripts for the 17.0.1.0.0